src/trace-junit.ts

```diff
@@ -245,8 +247,12 @@ export function traceTestSuite({
       );
       return testCaseTimeSec;
     });
-    testCasesTimeSec =
-      testCasesTimeSec || testCasesTimes.reduce((r, i) => r + i, 0);
+
+    // If testSuite exists with testCases
+    if (testCasesTimes) {
+      testCasesTimeSec =
```

Comment:
Careful with constant variables 😁 you probably need to declare it with `let`

Reply:
Uhm, It's defined in L231, is it also required in L253?

Reply:
Ah ah sorry!! I missed the plurals 🤦‍♂️ in L236 it defines `const testCaseTimeSec`